From b7a219a91dc1094022478dd9bc0f60275367ce25 Mon Sep 17 00:00:00 2001
From: David Tolnay <dtolnay@gmail.com>
Date: Sat, 24 Oct 2015 11:21:16 -0700
Subject: [PATCH] mktemp needs 6 or more X's (fix #1000)

mktemp(1) isn't posix specified, but it's reasonable to use
mktemp(3) to implement it which requires 6 trailing X's in
the format

Thanks @sthen
---
 tests/setup | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/setup b/tests/setup
index a6bc5f4..d64ba29 100755
--- a/tests/setup
+++ b/tests/setup
@@ -29,7 +29,7 @@ clean () {
     fi
 }
 trap clean EXIT
-d=`mktemp -d -t jqXXXXX || true`
+d=`mktemp -d -t jqXXXXXX || true`
 if [ -z "$d" ]; then
     echo "Your OS does not support mktemp(1) -d" 1>&2
     exit 1
-- 
2.14.1

